src/utils/useChannelFetch.ts

```diff
@@ -0,0 +1,54 @@
+import { useChannel } from "@storybook/manager-api";
+
+const FETCH_ABORTED = "ChannelFetch/aborted";
+const FETCH_REQUEST = "ChannelFetch/request";
+const FETCH_RESPONSE = "ChannelFetch/response";
```

Comment:
These events are duplicated, which could cause them to go out of sync.

A common practice is to have a `constants.ts` file, that's shared between manager/preview/server.

Reply:
Yes these should be in Storybook core/events eventually.